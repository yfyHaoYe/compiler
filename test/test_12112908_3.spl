int length;
struct array
{
  int index;
  char data;
};
int test_3()
{
  struct array arr_0;
  struct array arr_1;
  struct arr arr_2;
  length = 2;
  arr_0.index = 0;
  arr_0.data = 'a';
  arr_1.index = 1;
  arr_1.data = 'b';
  arr_0.index = 2;
  arr_0.data = 'v50';
}