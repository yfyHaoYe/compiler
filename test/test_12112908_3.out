Error type A at Line 18: Unknown characters ''v50''
